api_specs/v2/component_analyses.yaml

```diff
@@ -118,76 +304,98 @@ components:
           items:
             type: object
             properties:
-              vendor_cve_ids:
+              cve_id:
```

Comment:
I'm still not clean what are you going to send in this field. if it is a vendor id then string type is fine, else it must be a array because a synk vulnerability can have may CVES.

Reply:
Please let me know what is expected..?